### ISSUE TITLE
wave1 Update short links to mtl.how/menu and IIR measures

### ISSUE THREAD

**Issue by ritahitching** [lzim/teampsd#1289]
**Update short links to all projects and instruments on VA RedCap :** 

- [ ] Update **mtl.how** short links to all projects on VA RedCap. 
- [ ] Update **mtl.how**  short links to all survey instruments (e.g. mtl.how/menu only redirects to project MTL Menu (MTL Live). 
- [ ] Priority projects include:
- IIR Team Red Baseline
- IIR Team Blue Baseline
- IIR Team Red Follow-up
- IIR Team Blue Follow-up
- MTL Menue (MTL Live)

#483

**Comment by ritahitching** [comment]
@lzim Will you please:
- [ ] Create new short links for the VA RedCap projects for the IIT, which will be added to the survey links within the instruments of each project.  The the new priority projects are:
1. IIR Team Red Baseline
2. IIR Team Blue Baseline
3. IIR Team Red Follow-up
4. IIR Team Blue Follow-up
- [ ] Update the current mtl.how/menu shortlink to reflect the change of this project from Stanford RedCap to VA RedCap - project name is **MTL Menue (MTL Live)**
thank you

FYI @lijenn

**Comment by staceypark** [comment]
@ritahitching @lijenn 
Based on Monday 4/13 WG Lead meetings, this is being moved to wave1_teams. We don't need it until we start with sites.

**Comment by ritahitching** [comment]
Create an issue card for work to be done to create the Short links and cost out the time and assign.

**Comment by lzim** [comment]
@ritahitching @staceypark @lijenn 

I'm having a hard time understanding this thread:

1. I see from @staceypark that these tasks should all be assigned to **wave1_teams** - So, I've edited the title to reflect that change.

2. But, I don't understand @ritahitching message from 6 days ago:
> Create an issue card for work to be done to create the Short links and cost out the time and assign.

- @ritahitching  those, separate task cards were created and added to **wave1_teams** then shouldn't this card be cross-ref'd and closed?

**Comment by ritahitching** [comment]
@lzim 

- My note **Create an issue card for work to be done to create the Short links and cost out the time and assign** refers to creating an issue card to track the work of creating the shortlinks themselves (#1353 in wave1 epic).
- We discussed at the #hqhuddle 4/17  to create issue cards for work to be done associated with an issue, to make sure that people can protect the time to do work and not block the issue. 
- The current issue #1289 is to update the links within the VA RedCap projects and instruments themselves once the shortlinks have been made. 


**from @lzim **
> @ritahitching @staceypark @lijenn
> 
> I'm having a hard time understanding this thread:
> 
> 1. I see from @staceypark that these tasks should all be assigned to **wave1_teams** - So, I've edited the title to reflect that change.
> 2. But, I don't understand @ritahitching message from 6 days ago:
> 
> > Create an issue card for work to be done to create the Short links and cost out the time and assign.
> 
> * @ritahitching  those, separate task cards were created and added to **wave1_teams** then shouldn't this card be cross-ref'd and closed?

**Comment by lzim** [comment]
Thanks @ritahitching !
- I couldn’t tell them apart based on the titles and details in the checklists, until I read this in your last comment above (Have I got it? 😃):
> ...create...shortlinks
> ...update links _within_ VA RedCap...

Since #1353 is blocking #1289, I added the **blocking dependency** following the SOP.

For a first step in this checklist and in #1289
- Does it make sense to create a template in the appropriate **master_document_card** for **RedCap** in the **documnet_tracker** of the list of links that will need to be 
a) created and 
b) updated within RedCap?

**Comment by staceypark** [comment]
@lzim @ritahitching 
1. I'm not sure why this got assigned a May due date, given that this is part of wave1. We had discussed in the HQ huddle that it would be fine to pull things early out of the May epic if April items were done, but Wave1 items specifically do not need to be done until we start with teams.

2. Thanks Rita for trying to follow the SOP and make sure there are task cards for every piece of work! But, I'm not sure why there are multiple task cards for this.
- [ ] There needs to be **one** task card that includes all the links Lindsey needs and then
- [ ] Lindsey needs to create the short links using ReBrandly.

Rita, if I'm reading the thread correctly it looks like you scoped out work for links to be updated _within_ RedCap? This was neither part of training or any process that I've covered. 

I don't believe there is a place to update links within RedCap unless I'm missing something. Can you point me to where this process would happen?
> - The current issue #1289 is to update the links within the VA RedCap projects and instruments themselves once the shortlinks have been made.

**Comment by ritahitching** [comment]
@lijenn this is issue is blocked by the #1192 and the MVP process.
If you read the thread you will find details of the links that need to be created.
I recommend we keep this issue open.

**Comment by lzim** [comment]
@lijenn @staceypark 

Is this something we still need? I'm struggling to track the current status?

**Comment by lijenn** [comment]
This is definitely a duplicate of #1353 which is to create/update shortlinks for all va redcap surveys. Updated the issue card there and closing this one.
